tests/ttnn/unit_tests/operations/test_core.py

```diff
@@ -439,3 +439,37 @@ def test_create_sharded_memory_config(device, shape, strategy, orientation, core
 
     passing = torch.equal(input_data, output_data)
     assert passing
+
+
+@pytest.mark.parametrize(
+    "shape, strategy, orientation, core_grid",
+    [
+        ([1, 1, 16, 32], ttnn.ShardStrategy.WIDTH, ttnn.ShardOrientation.ROW_MAJOR, ttnn.CoreGrid(y=2, x=1)),
+        ([1, 1, 32, 16], ttnn.ShardStrategy.HEIGHT, ttnn.ShardOrientation.ROW_MAJOR, ttnn.CoreGrid(y=2, x=1)),
+        ([1, 1, 64, 16], ttnn.ShardStrategy.HEIGHT, ttnn.ShardOrientation.ROW_MAJOR, ttnn.CoreGrid(y=2, x=1)),
+        ([1, 1, 16, 16], ttnn.ShardStrategy.HEIGHT, ttnn.ShardOrientation.ROW_MAJOR, ttnn.CoreGrid(y=2, x=1)),
+    ],
```

Comment:
some tests to try out (a shape in group norm ) does 32x320 width sharding to 8 cores pass?